app/controllers/users_controller.rb

```diff
@@ -5,4 +5,8 @@ def homepage
     @user = current_user
     @topics = Topic.all
   end
+
+  def edit_first_login
```

Comment:
You shouldn't have a method for updating a particular attribute of your model.
Following the standard CRUD actions, you should replace this method for a `update` action in your controller.

Remember to use strong params and the correct HTTP verb